nelua/cdefs.lua

```diff
@@ -189,9 +192,90 @@ cdefs.target_info_code = [[
 #if defined(__ANDROID__)
   is_android = true;
 #endif
+#if defined(APPLE_IOS) || defined(IOS) || defined(__IPHONEOS__) || defined(TARGET_OS_SIMULATOR) || defined(__IOS__) || defined(__APPLE_IOS__) || defined(IOS_OBJ_C) || defined(TARGET_OS_IOS)
```

Comment:
This line is too long, `make check` will fail, it warns about lines longer then 120 columns, same apply for other long lines found in the PR. Please split the lines using \, like:

```c
if defined(APPLE_IOS) || defined(__APPLE_IOS__) || \
   defined(TARGET_OS_IOS) || defined(TARGET_OS_SIMULATOR) || \
   defined(__IOS__)  || defined(IOS)
```

Also I think `APPLE_IOS` and `IOS` could be removed, I would that most defines without any `_` to be removed if possible, I will comment more about ones below.

Reply:
Done this now via commit to the patch! :D